src/pages/Contributing/Glossary/index.md

```diff
@@ -1,4 +1,97 @@
 ---
 title: 'Glossary'
 order: 2
----
\ No newline at end of file
+---
+
+> This is a list of terms we use currently in our routine with Mozaic, we’re adding here these terms + definitions just in case you are wondering “what the heck does it mean?” well, here you’ll have the proper info! :)
```

Comment:
This is a list of terms we use in our routine with Mozaic. We are adding these terms and definitions just in case you are wondering "what the heck does it mean ?" Well, you will find the proper piece of informations on this page :) !